ref/general/federated-registries.adoc

```diff
@@ -0,0 +1,119 @@
+
+[source,java]
+----
+<server description="Federation">
+    <featureManager>
+        <feature>appSecurity-2.0</feature>
+        <feature>servlet-3.0</feature>
+	 <feature>ldapRegistry-3.0</feature>
+    </featureManager>
+
+    <basicRegistry id="basic" realm="SampleBasicRealm">
+        <user name="admin" password="password" />
+        <user name="user1" password="password" />
+            <user name="user2" password="password" />
+        <group name="memberlessGroup" />
+        <group name="adminGroup">
+            <member name="admin"/>
+        </group>
+        <group name="users">
+            <member name="user1"/>
+            <member name="user2"/>
+        </group>
+    </basicRegistry>
+
+    <ldapRegistry id="LDAP1" realm="SampleLdapIDSRealm" host="LDAPHOST1.ibm.com" port="389" ignoreCase="true"
+	baseDN="o=ibm,c=us"
+	ldapType="IBM Security Directory Server"
+	searchTimeout="8m"
+	recursiveSearch="true">
+    </ldapRegistry>
+
+    <federatedRepository>
+        <primaryRealm name="PrimaryRealm" allowOpIfRepoDown="true">
+            <participatingBaseEntry name="o=SampleBasicRealm"/>
+            <participatingBaseEntry name="o=ibm,c=us"/>
+        </primaryRealm>
+   </federatedRepository>
+</server>
+----
+
+== Federation of basic and custom registries
+
+To federate two or more basic registries, you must enable both link:/docs/ref/feature/#appSecurity-3.0.html[the Application Security feature] and the Federated User Registry feature, as shown in the following example.
+
+[source,java]
+----
+<server description="Federation">
+    <featureManager>
+        <feature>appSecurity-2.0</feature>
+        <feature>federatedRegistry-1.0</feature>
+    </featureManager>
+
+    <basicRegistry id="basic1" realm="SampleBasicRealm1">
+        <user name="admin" password="password" />
+        <user name="user1" password="password" />
+            <user name="user2" password="password" />
+        <group name="memberlessGroup" />
+        <group name="adminGroup">
+            <member name="admin"/>
+        </group>
+        <group name="users">
+            <member name="user1"/>
+            <member name="user2"/>
+        </group>
+    </basicRegistry>
+
+    <basicRegistry id="basic2" realm="SampleBasicRealm2">
+        <user name="user3" password="password123" />
+        <user name="user4" password="password123" />
+        <group name="memberlessGroup2" />
+        <group name="users2">
+            <member name="user3"/>
+            <member name="user4"/>
+        </group>
+    </basicRegistry>
+</server>
+----
+
+You can add basic or custom registries to a custom federated repository by defining the `participatingBaseEntry` name field in the `federatedRepository` element. The participating base entry for a custom or basic registry is the organization attribute set to the realm name of that registry. In the following example shows the federation of a basic registry and a custom registry:
```

Comment:
Change,
"In the following example shows the federation of a basic registry and a custom registry"
to,
"The following example shows federation of a basic registry and a custom registry"